setup.py

```diff
@@ -47,5 +47,6 @@
     cmdclass=cmdclass,
     python_requires='>=3.5',
     zip_safe=False,
+    install_requires=['sphinx-astropy-theme'],
```

Comment:
You don't actually *need* the theme to use helpers... I can understand why you would want to keep it though.

Reply:
You could make the `conf.py` fall back to the default sphinx `alablaster` theme if the astropy one isn't installed.

Reply:
I don't think we should have a fallback as I think for now users won't want the theme to suddenly change, and it's easier to tell users how to install the theme if there is an error.